MangaManager/src/MetadataManager/MetadataManagerGUI.py

```diff
@@ -456,7 +419,14 @@ def process_fetch_online(self):
             mb.showwarning("Not a valid series name", "The current series name is empty or not valid.")
             self.log.info("Not a valid series name - The current series name is empty or not valid.")
             return
-        cinfo = self.fetch_online(series_name)
+
+        temp = ComicInfo()
+        temp.year = self.widget_mngr.get_widget("Year").get()
+        temp.number = self.widget_mngr.get_widget("Number").get()
+        temp.volume = self.widget_mngr.get_widget("Volume").get()
```

Comment:
These should be "keep_original_value". 
Feel free to ignore this for now i don't recall if the fetch metadata applies to multiple selected files.
If it does it def need to be "keep_original_value"